src/app/main/component/events/services/events.service.ts

```diff
@@ -133,6 +133,14 @@ export class EventsService implements OnDestroy {
     return this.http.get<EventResponse>(`${this.backEnd}events/${eventId}`);
   }
 
+  getIsLikedByUser(eventId: number): Observable<boolean> {
+    return this.http.get<boolean>(`${this.backEnd}events/${eventId}/likes`);
+  }
+
+  postToggleLike(eventId: number): Observable<any> {
+    return this.http.post(`${this.backEnd}events/${eventId}/like`, {});
+  }
+
```

Comment:
I'm sorry, my bad, could you update `getAllAttendees` as using `http.get<any>` is a bad practice.
Please change it to http.get<EventAttender[]>. Thank you very much